StringCalculator.Tests/StringCalculatorTests.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using Xunit;
+
+namespace StringCalculator.Tests
+{
+    public class StringCalculatorShould
```

Comment:
`Should` at the end of the class name is not normal. Usually, it will be class_nameTest
`StringCalculatorTest`